File: framework/src/main/java/dto/EtatServer.java
package dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class EtatServer /*implements Serializable*/ {

    private Long id;
    private Double tauxEchecPing;
    private Double tauxEchecHttp;
    private Double tauxEchecTelnet;
    private Double etatConnexion;
    private Date dateAnalyse;
    private List<Notification> notifications;
    private ServerHote serverHote;

}

File: monitor-view/src/app/shared/entities/EtatServer.ts
import {Moment} from 'moment';

export class IEtatServer {
  checked?: boolean;
  dateAnalyse?: Moment;
  id?: number;
  tauxEchecHttp?: number;
  tauxEchecPing?: number;
  tauxEchecTelnet?: number;
  etatConnexion?: number;
}
export class EtatServer implements IEtatServer{
  checked: boolean;
  dateAnalyse: Moment;
  id: number;
  tauxEchecHttp: number;
  tauxEchecPing: number;
  tauxEchecTelnet: number;
  etatConnexion?: number;
}
